extension/js/common/platform/store/abstract-store.ts

```diff
@@ -13,7 +13,7 @@ import { StoredAdminCode } from './global-store.js';
 type SerializableTypes = FlatTypes | string[] | number[] | boolean[] | DomainRulesJson;
 export type StorageType = 'session' | 'local';
 export type FlatTypes = null | undefined | number | string | boolean;
-type Storable = FlatTypes | string[] | KeyInfo[] | Dict<StoredAdminCode>
+type Storable = FlatTypes | string[] | StoredKeyInfo[] | KeyInfoWithIdentity[] | Dict<StoredAdminCode>
```

Comment:
For now, I started saving `KeyInfoWithIdentity` for newly saved or updated keys. In the future we'll do a migration to update all `StoredKeyInfo` to `KeyInfoWithIdentity` format, and start expecting `KeyInfoWithIdentity` to come out of storage. That will clean up the code a bit.

Reply:
#4378